types/consumer-data-standards/package.json

```
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/consumer-data-standards",
-    "version": "7.2.9999",
+    "version": "7.3.0",
```

```suggestion
    "version": "7.3.9999",
```

This suffix is required (and will not be used when publishing).

https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library